Lib/subprocess.py

```diff
@@ -978,6 +979,19 @@ def __init__(self, args, bufsize=-1, executable=None,
 
             raise
 
+    def __repr__(self):
+        max_args_length = 30
```

Comment:
IMO this is too small. Consider increasing so that the full repr is up to 80 characters long.

Also, consider removing the class's module from the repr to allow increasing this further.